### PR TITLE
Feature/support dotnet standard 2

### LINKED ISSUE #10: Target netstandard2.0
...so that users are not forced to pull in a lot of references due to only supporting netstandard 1.1.

### PR DESCRIPTION
**Pull request by ike86**
Attempts to resolve #10

### CODE REVIEW

File: test/Xunit.Categories.Test/Scratch.cs [hunk 1]
```diff
@@ -1,12 +1,9 @@
 ﻿using System;
```
Comment (ike86) [inline test/Xunit.Categories.Test/Scratch.cs:1]: All changes in this file are unrelated. Sorry about that.

**Comment by brendanconnolly** [comment]
thanks for the update I'll update the nuget package shortly!
